contributors/devel/cherry-picks.md

```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The Patch Release Manager
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
+   * The Patch Release Manager for the target release branch is responsible for
+   applying the `cherrypick-approved` label. You can find the release team contacts
```

Comment:
IMO you can drop "You can find the release team contacts..." if you link Patch Release Manager directly.  That description includes a link to all release / patch release branch managers

Reply:
Might still want a few words around how the person doing the role changes in each release and to look at the release vX.Y specific table to find the right person.

Reply:
@tpepper - thanks to @spiffxp 's link in comment above, I think the flow of information is relatively clear and obvious to infer.